device/hugepage.h

```diff
@@ -27,4 +27,4 @@ std::string find_hugepage_dir(std::size_t pagesize);
 // Today we assume there's only one pipeline running within the system.
 // One hugepage per device such that each device gets unique memory.
 int open_hugepage_file(const std::string &dir, chip_id_t physical_device_id, uint16_t channel);
-}
\ No newline at end of file
+}
```

Comment:
I think that one tool tries to enforce no newlines at the end of the file, and another does the opposite.
If you continue seeing this let me know so I can investigate. No action here